include/integration/simple_MC_integration.hpp

```diff
@@ -179,7 +201,8 @@ template
     typename NT = NT,
     typename Functor
 >
-NT simple_mc_integrate (Functor Fx, 
+NT simple_mc_integrate (Functor Fx,
```

Comment:
Since you are changing this file, could you please remove the empty space between function name and `(` so that all the functions have the same coding style.